Nitrox.Launcher/Models/Design/MultiDataTemplate.cs

```diff
@@ -0,0 +1,46 @@
+extern alias JB;
+using System.Collections.Generic;
+using Avalonia.Collections;
+using Avalonia.Controls;
+using Avalonia.Controls.Templates;
+using Avalonia.Markup.Xaml.Templates;
+using Avalonia.Metadata;
+using JB::JetBrains.Annotations;
+
+namespace Nitrox.Launcher.Models.Design;
+
+/// <summary>
+///     Selects a <see cref="DataTemplate" /> based on its <see cref="DataTemplate.DataType" />.
+/// </summary>
+public class MultiDataTemplate : AvaloniaList<DataTemplate>, IDataTemplate
```

Comment:
Didn't look at how it's used so I might be missing a point, but what's the difference between that and a DataTemplateSelector ?

Reply:
WPF has that but Avalonia doesn't provide one by default, see bottom of page here <https://docs.avaloniaui.net/docs/get-started/wpf/datatemplates#data-template-selector>